api_fhir/permissions.py

```diff
@@ -1,17 +1,83 @@
+from claim import ClaimConfig
```

Comment:
Is there a difference as here is claim and all the others have '.apps' in module name?

Reply:
the __init__.py in claim makes it work, but indeed, better be homogenous,
Thx